src/app/api/user/route.ts

```
@@ -0,0 +1,49 @@
+// NOTE: 전체 유저 불러오는 엔드포인트
+export async function GET(req: NextRequest) {
+  try {
+    await dbConnect();
+
+    const rawPage = parseInt(req.nextUrl?.searchParams.get('page') ?? '1', 10);
+    const rawLimit = parseInt(
+      req.nextUrl?.searchParams.get('limit') ?? '10',
+      10
+    );
+
+    if (isNaN(rawPage) || isNaN(rawLimit)) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)